src/code/models/link-manager.coffee

```diff
@@ -333,9 +328,12 @@ module.exports   = class LinkManager
     linkExports = for key,link of @linkKeys
       link.toExport()
     imageMetadata = {}
-    _.forEach palette, (node) =>
-      if @imageMetadataCache[node.image]
-        imageMetadata[node.image] = @imageMetadataCache[node.image]
+    # The pallete manager is curently in charge of trackign image meta-data
+    PaletteManager   = require "../models/palette-manager"
```

Comment:
I should address this now, its an ugly hack.  Link-manager was requiring palette-manager -- and palette-manager was requiring the link-manager.  This defers the require until its needed.  I don't like it. Circular are a code smell to me. 

We only need this because the link-manager is serializing everything. This is actually why the palette manager required the link-manager in the first place.  

I think the right thing to do would be to create a special controller for Loading & Saving.  I think I can do that as part of this work here:  https://www.pivotaltracker.com/story/show/100146458